notebooks/3_subset_data.ipynb

```diff
@@ -0,0 +1,1358 @@
+{
```

Comment:
<p>I think it's worth mentioning explicitly that each condition must be surrounded by parentheses, and you have to use <code>|</code> and <code>&amp;</code> (you can't use <code>or</code> or <code>and</code>).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>This is indeed a typical problem. Added as a note to the tutorial</p><p> </p>